index.js

```diff
@@ -0,0 +1,280 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let get = document.querySelector("#get");
+    get.addEventListener("click", getTwoPoke);
+    
+    let fight = document.querySelector("#fight");
+    fight.addEventListener("click",battle);
+}
+)
+
+const getTwoPoke = ()=>{
+    return getPokemon2(), getPokemon()
+}
+
+const getPokemon = async () => {
+    try {
+        let random = Math.floor((Math.random() * 151) + 1);
+        let res = await axios.get(`https://cors-anywhere.herokuapp.com/https://pokeapi.co/api/v2/pokemon/${random}/`);
+        // debugger;
+        let pokeName=res.data.name;
+        let pokeImg=res.data.sprites.front_default;
+        let pokeHP =res.data.stats[5].base_stat;
+
+        
+        let newMove=shuffleMove(res.data.moves);
+        
+        let move1 = newMove[0].move.name;
```

Comment:
You can mave a for loop that goes up to 4 and makes it for you instead of going though yourself